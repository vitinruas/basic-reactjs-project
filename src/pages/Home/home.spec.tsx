import { render } from '@testing-library/react'
import { Home } from '.'

describe('<Home />', () => {
  test('Should render search', () => {
    render(<Home />)
  })
})
