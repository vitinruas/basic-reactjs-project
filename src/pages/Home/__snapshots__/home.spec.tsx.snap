// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Should match snapshot 1`] = `
<div
  class="container"
>
  <div
    class="search-container"
  >
    <input
      class="search-input"
      placeholder="Type to look for something"
      type="text"
      value=""
    />
  </div>
  <div
    class="posts"
  />
  <button
    class="button"
    disabled=""
  >
    Load more Posts
  </button>
  <span>
    There is not anything :(
  </span>
</div>
`;
